indexer/services/ender/src/scripts/dydx_update_perpetual_position.sql

```diff
@@ -0,0 +1,61 @@
+CREATE OR REPLACE FUNCTION dydx_update_perpetual_position(
+    subaccount_uuid uuid,
+    perpetual_id bigint,
+    side text,
+    size numeric,
+    price numeric
+) RETURNS perpetual_positions AS $$
+DECLARE
+    perpetual_position_record RECORD;
+    sum_open numeric;
+    entry_price numeric;
+    sum_close numeric;
+    exit_price numeric;
+BEGIN
+    -- Retrieve the latest perpetual position record
+    SELECT * INTO perpetual_position_record
+    FROM perpetual_positions
+    WHERE "subaccountId" = subaccount_uuid
+      AND "perpetualId" = perpetual_id
+    ORDER BY "createdAtHeight" DESC
+    LIMIT 1;
```

Comment:
You wouldn't have known about this, but I'm pushing a PR right now to order by `openEventId` instead of `createdAtHeight`, its a bug that we just uncovered today and track here. Please order by `openEventId`. https://linear.app/dydx/issue/IND-485/inconsistent-perpetual-position-querying

Reply:
going to address it in a separate PR since it breaks order handler unit tests/it's not part of deleveraging.